apps/src/templates/progress/progressHelpers.js

```diff
@@ -156,6 +156,20 @@ export function lessonHasLevels(lesson) {
   return !!lesson.levels?.length;
 }
 
+/**
+ * Checks if there are any levels in the unit
+ * @param {array} list of lessons
+ * @returns {bool} if the unit has any levels
+ */
+export function unitHasLevels(lessons) {
+  lessons.forEach(lesson => {
+    if (lessonHasLevels(lesson)) {
+      return true;
+    }
+  });
+  return false;
```

Comment:
I believe this will do the trick here. Is anything using this function though?

```suggestion
  return lessons.some(lessonHasLevels);
```

Reply:
oh dang thanks for catching this! A remnant from a past iteration/approach. Removed:)